packages/app/src/components/DBRowTable.tsx

```diff
@@ -693,7 +693,8 @@ function mergeSelectWithPrimaryAndPartitionKey(
     .split(',')
     .map(k => extractColumnReference(k.trim()))
     .filter((k): k is string => k != null && k.length > 0);
-  const primaryKeyArr = primaryKeys.split(',').map(k => k.trim());
+  const primaryKeyArr =
```

Comment:
good call on this. this doesn't rule out the string with spaces. how about
```
primaryKeys.trim() !== "" ? primaryKeys.split(',').map(k => k.trim()) : [];
```

Reply:
cool, I will use this. Thanks!